app/Http/Controllers/DependencyController.php

```diff
@@ -38,7 +44,7 @@ public function getAllDependencies() : JsonResponse
 
             return [
                 'key' => $dependency->key,
-                'uri' => sprintf('%s/%s', config('app.url'), $dependency->uri),
+                'uri' => sprintf('%s/dependency/%d', config('app.url'), $dependency->id),
```

Comment:
```suggestion
                'uri' => "{config('app.url')}/dependency/{$dependency->id}",
```

Reply:
String interpolation in PHP only works like this for variables - it doesn't work for method calls like `config` :(